Views/messages.erb

```diff
@@ -0,0 +1,25 @@
+Messages:<br>
+   <a href="new_peep">New peep!</a> <br>
+<br>
+  Search:
+  <form action="/messages" method="get">
+    <input type="text" name="find">
+    <input type="submit" id="submit">
+  </form>
+<br>
+<% if @reversed %>
+  <a href="/messages">Sort by new.</a>
+  <ul>
+    <% @results.reverse_each do |element| %>
+      <li> <%= "(" << element["date"][5..18] << "):  " %> <%= element["message"] %> </li> 
+    <% end %>
+  </ul>
+
+<% else %>
+  <a href="/messages?reversed=true">Sort by old.</a>
+  <ul>
+    <% @results.each do |element| %>
+      <li> <%= "(" << element["date"][5..18] << "):  " %> <%= element["message"] %> </li> 
+    <% end %>
+  </ul>
+<% end %>
```

Comment:
This works well, quite simple and elegant but could the code be written in a different file (i.e. Chitter.rb in /lib folder) following the MVC pattern? i.e. the erb file could contain only the minimum code required to display the message.